GG/GG/BrowseInfoWnd.h

```diff
@@ -158,7 +158,7 @@ class GG_API TextBoxBrowseInfoWnd : public BrowseInfoWnd
     Clr                     m_border_color;
     unsigned int            m_border_width;
     X                       m_preferred_width;
-    TextControl*            m_text_control;
+    std::shared_ptr<TextControl>            m_text_control;
```

Comment:
Can you fix or work around the layout issues caused by this sort of simple find-replace?